frontend/packages/operator-lifecycle-manager/src/components/create-operand.tsx

```diff
@@ -675,465 +919,376 @@ export const CreateOperandForm: React.FC<CreateOperandFormProps> = ({
     return null;
   };
 
-  const getGroupName = (group, groupType) => {
```

Comment:
Replaced this with a similar helper function that uses regex to parse group name and type from the full descriptor.